e2e/scenes/dashboard-suite/new-interval-variable.spec.ts

```diff
@@ -41,8 +40,8 @@ describe('Variables - Interval', () => {
 
     e2e.components.RefreshPicker.runButtonV2().click();
 
-    e2e.pages.Dashboard.SubMenu.submenuItemValueDropDownValueLinkTexts('10s').click();
-    e2e.pages.Dashboard.SubMenu.submenuItemValueDropDownOptionTexts('1h30m').click();
+    e2e.pages.Dashboard.SubMenu.submenuItemLabels('Variable under test').next().should('have.text', `10s`).click();
```

Comment:
Do we have any way to avoid `next`? Depending on the component structure could be flaky

Reply:
I am adding additional data-testids and refactoring this in https://github.com/grafana/grafana/issues/87724